### PR TITLE
fix: Toast Duration

### PR DESCRIPTION
**Pull request by AnoopkRajan**
Fixed: #79
Changes: Toast duration has been reduced.

### CODE REVIEW

**Comment by AnoopkRajan** [comment]
@yashk2000 
Thanks for the review.
Working on the build fail.
format will be updated next PR onward.

File: app/src/main/java/org/amfoss/templeapp/activities/DeleteData.java [hunk 1]
```diff
@@ -43,13 +43,13 @@ public void radioGroupOnCheckedChanged(RadioGroup group, int checkedId) {
...
                     flag = 0;
-                    Toast.makeText(getBaseContext(), getString(R.string.delete_donated), Toast.LENGTH_LONG)
+                    Toast.makeText(getBaseContext(), getString(R.string.delete_donated), Toast.LENGTH_SHORT)
                             .show();
                     break;
                 case R.id.radio_pooja:
                     binding.totalView.setVisibility(View.VISIBLE);
                     flag = 1;
-                    Toast.makeText(getBaseContext(), getString(R.string.delete_pooja), Toast.LENGTH_LONG)
+                    Toast.makeText(getBaseContext(), getString(R.string.delete_pooja), Toast.LENGTH_SHORT)
```
Comment (Chromicle) [inline app/src/main/java/org/amfoss/templeapp/activities/DeleteData.java]: Change to `getBaseContext` instead of `getApplicationContext` as this is that preferred way since this Context lives untill Application shuts down.
I am seeing so many unused changes please change that

Comment (AnoopkRajan) [reply app/src/main/java/org/amfoss/templeapp/activities/DeleteData.java]: @Chromicle 
Can you please highlight the unused changes . 
So that i can know the specifics.

Comment (AnoopkRajan) [reply app/src/main/java/org/amfoss/templeapp/activities/DeleteData.java]: Should i also change to getBaseContext instead of getApplicationContext in MainActivity ?

File: MyApplication/app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,29 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.0"
+    defaultConfig {
```
Comment (Chromicle) [inline MyApplication/app/build.gradle]: This is not templeApp gradle file you created some thing application named `MyApplicattion`
Please revert all these

Comment (AnoopkRajan) [reply MyApplication/app/build.gradle]: @Chromicle 
Yes , I overlooked this fault .
working on it.

File: gradle.properties [hunk 3]
```diff
@@ -19,4 +19,5 @@ org.gradle.daemon=true
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+org.gradle.jvmargs=-Xmx1024m
```
Comment (immadisairaj) [inline gradle.properties]: revert

Comment (AnoopkRajan) [reply gradle.properties]: @immadisairaj 
I have reverted it back.
But i am getting additional scroll fix codes.
I dont kno whether these are changed codes or previous codes.

**Comment by Chromicle** [comment]
> @AnoopkRajan Also can you update the pr template.

@AnoopkRajan  what he ment is change in Github PR template not in `pull_request_template.md`

File: gradle.properties [hunk 4]
```diff
@@ -19,4 +19,4 @@ org.gradle.daemon=true
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
```
Comment (immadisairaj) [inline gradle.properties]: this is not a change. then why is it here. can you rectify that

File: pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Related Issue
```
Comment (immadisairaj) [inline pull_request_template.md]: this is not what I meant. You have to fill the PR template for the Pull Request

Comment (immadisairaj) [reply pull_request_template.md]: revert this

**Comment by immadisairaj** [comment]
> Fixed Toast Duration issue .

I meant change this according to the PR template

File: gradle.properties (hunk 4 above)
Comment (AnoopkRajan) [reply gradle.properties]: Reverted

**Comment by AnoopkRajan** [comment]
> > Fixed Toast Duration issue .
> 
> @AnoopkRajan I meant change this according to the PR template

Have added the PR template.

File: .github/PULL_REQUEST_TEMPLATE [hunk 6]
```diff
@@ -1,5 +0,0 @@
-Fixed #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
```
Comment (yashk2000) [inline .github/PULL_REQUEST_TEMPLATE]: Why did you delete this file? Revert this please.

File: gradle.properties (hunk 4 above)
Comment (immadisairaj) [reply gradle.properties]: it is not yet reverted.

**Comment by yashk2000** [comment]
@AnoopkRajan please revert changes to the pr template. Rest seems to be fine.

**Comment by AnoopkRajan** [comment]
> @AnoopkRajan please revert changes to the pr template. Rest seems to be fine.

yes .... i am working on it .

**Comment by vchrombie** [comment]
Thanks, @AnoopkRajan for the PR. :slightly_smiling_face:
